src/types/model.ts

```diff
@@ -183,14 +183,16 @@ export interface StudyTerm {
   dataTypes: string[]
 }
 
-interface DataUseTerm {
+export interface DataUseTerm {
   code: string
   description: string
+  manualReview?: boolean
+  type?: string
 }
 
-interface DataUseSummary {
-  primary: DataUseTerm[]
-  secondary: DataUseTerm[]
+export interface DataUseSummary {
+  primary?: DataUseTerm[]
```

Comment:
I thought the primary should never be optional?

Reply:
I believe that is true, will take a second look at why I made this change.